lti_consumer/models.py

```diff
@@ -296,3 +315,17 @@ def __str__(self):
 
     class Meta:
         unique_together = (('line_item', 'user_id'),)
+
+
+@receiver(post_save, sender=LtiAgsScore)
```

Comment:
Create a `signals.py` file and move this code there.

Don't forget to import the signals on `app.py` in the `ready` method, otherwise they won't work properly in the platform.

Reply:
Here I've added a receiver and didn't create any custom signal so I don't have to add them into `app.py` `ready` method.